prime-router/settings/organizations-prod.yml

```diff
@@ -131,7 +131,6 @@
       deidentify: false
       translation:
         type: HL7
-        schemaName: fl/fl-covid-19
         useBatchHeaders: true
```

Comment:
Removing the schema names in prod. We don't use them there. Just muddies the waters.

Reply:
Thanks! - going forward, only "CUSTOM" needs a schema name?

Reply:
Correct.